### PR TITLE
Adds NFS dynamic provisioning

### PR DESCRIPTION
**Pull request by firemound**

### CODE REVIEW

File: kubeinit/roles/kubeinit_nfs/tasks/main.yml [hunk 1]
```diff
@@ -69,3 +69,244 @@
...
+  tags:
+    - provision_libvirt
+
+#
+# add nfs dynamic provisioning
+#
+- name: add security context constraint for nfs provisioner
+  shell: |
+    cat << EOF > ~/nfs-scc.yaml
+    apiVersion: security.openshift.io/v1
```
Comment (firemound) [inline kubeinit/roles/kubeinit_nfs/tasks/main.yml:136]: @ccamacho it looks like the nfs role is referenced outside of okd. I'm not sure if there's a common resource definition that can be used here. I'll have to dig around and test unless you're aware of a solution for this type.

**Comment by firemound** [comment]
@ccamacho ouch, we got pull rate limited by docker hub it looks like...

**Comment by ccamacho** [comment]
@firemound sorry for not getting back to you before, but an upgrade in podman broke everything.. took me 2 days to fix it but we are back on track, the changes looks good. but they wont get merged because of the commit titles don't match the lint check.
Can you put them like this?
refactor: this is a refactor

For the docker pull issue, now we have two variables to configure it :) like https://github.com/Kubeinit/kubeinit/blob/master/kubeinit/hosts/common.yml#L3-L4

**Comment by firemound** [comment]
@ccamacho you are very responsive and I have no complaint, thank you. I don't mind amending the commits, but I will not be able to do so without altering the commit history. I think this is considered bad practice when/if the history ever gets re-based. I'll push those updates today.

**Comment by ccamacho** [comment]
@firemound yeah I dont think that would be a problem, as long as the forced push is in the forks there are no problems. What is important is that once the code is merged, then the history on the main repo should not be altered. Direct merges and force is disabled in the repo.

File: kubeinit/roles/kubeinit_nfs/tasks/main.yml (hunk 1 above)
Comment (ccamacho) [reply kubeinit/roles/kubeinit_nfs/tasks/main.yml:136]: no worries with this, if the code passes the lint checks Ill merge it and iterate over.

**Comment by firemound** [comment]
Gah, I screwed up re-basing after merging in latest from master. I'm going to open up another PR.

**Comment by firemound** [comment]
Corrected PR is here: #123

**Pull request closed without merging**